### DISCUSSION TITLE
Skupper Router service

### DISCUSSION THREAD

**Discussion by jhuynh08** [skupperproject/skupper-router#295] [Q&A] [answered]
Hello,

Is it possible to change the skupper router service on kubernetes to deploy as clusterIP? I want to use clusterIP with kubernetes ingress instead of having to provision a new load-balancer for each skupper deployments.

**Comment by jiridanek** [comment]
I think you can use NodePort. There seems to be a test for that in skupper at

https://github.com/skupperproject/skupper/blob/954e2242c4abb98361090b41d0de4f6afe5312b1/test/integration/acceptance/custom/basic/basic.go#L199-L200

Hopefully somebody will be able to give more complete answer.

**Comment by kgiusti** [comment]
@grs @ted-ross would know for sure.

**Comment by grs** [comment, marked as answer]
You can specify --ingress=none (but then it won't be accessible outside the cluster)
